.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -34,24 +34,25 @@ env:
   USE_CUDA: 1
 {%- endif %}
 
+
 concurrency:
   group: !{{ build_environment }}-${{ github.event.pull_request.number || github.sha }}
   cancel-in-progress: true
 
 jobs:
   build:
     runs-on: "windows.4xlarge"
+    defaults:
+      run:
+        working-directory: pytorch-${{ github.run_id }}
```

Comment:
even with its new working-directory, let's also add the cleanup script. Understandable that scale-down can do some cleanup, but there's a small chance that the runner can run for a long time and can run out of disk

Reply:
Fair enough, I'll add that in